scripts/eslint-pre-commit

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-git stash -q --keep-index
-
-# Test prospective commit
-git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

Comment:
I haven't looked all that much into understanding and fixing this, I admit, but I just know the stashing bit goes awry all the time, leaving me with unstaged files put in the stash, and then when I `stash pop` I sometimes get a merge conflict I need to deal with, quite needlessly. Compared to the minor nuisance of sometimes of the linting failing on some unfinished work, I'll take the latter any day.